test/ssltest.c

```diff
@@ -1528,10 +1620,12 @@ int main(int argc, char *argv[])
      */
     SSL_CTX_set_security_level(c_ctx, 0);
     SSL_CTX_set_security_level(s_ctx, 0);
+    SSL_CTX_set_security_level(s_ctx2, 0);
```

Comment:
This file would now benefit from a common `setup_server_ctx()` method or some such. But if it looks like it'd be a lot of work, don't bother.